setup.py

```diff
@@ -37,6 +37,7 @@ def get_test_suite():
     },
     scripts=[
         'scripts/aclshow',
+        'scripts/arp-helper-start',
```

Comment:
rename to arp-helper is better, we could have arp-helper start ip address or arp-helper stop later.